consume-lsp

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+# This script is meant to be run after the Flux release script has
+# opened a PR in the LSP, and that PR has merged.
+
+# It should handle the rest of the release process, with minimal 
+# babysitting by an engineer.
+
+# If any command fails, exit the script
+set -e
+
+# Check if hub is installed
+if ! command -v hub &> /dev/null
+then
+    echo "hub is not installed. exiting"
+fi
+
+# Make a temporary directory where we can clone fresh copies of all the necessary repos
+TEMPDIR=$(mktemp -d -t lsp-release.XXXX)
+echo "Using ${TEMPDIR}"
+cd $TEMPDIR
+
+# Cut down on code duplication
+function tmp_clone() {
+	git clone git@github.com:influxdata/$1.git &> /dev/null
+	echo "$(pwd)/$1"
+}
+
+function extract_version() {
+	v=$(echo "$1" | grep -Po -m 1 '\d+\.\d+\.\d+')
+	echo $v
+}
+
+FLUX_DIR=$(tmp_clone flux)
```

Comment:
Realizing now that it's probably not necessary to clone `flux`, since this script never does anything with it.

Reply:
We use it to get the flux version, but we don't do anything with it (yet). If we decide we don't actually need that, we can save some time by not cloning flux.